src/gridstack.js

```diff
@@ -777,19 +787,25 @@
                 }
                 self.container.addClass(self.opts.oneColumnModeClass);
                 oneColumnMode = true;
+                var oneColumnModeSort = self.opts.oneColumnModeSort;
+                if (oneColumnModeSort !== 'none') {
+                    var dim = oneColumnModeSort.includes('column') ? -1 : 1;
+                    var dir = dim > 0 && self.opts.rtl ? -1 : 1;
+                    dir = oneColumnModeSort.includes('reverse') ? dir * -1 : dir;
```

Comment:
code could use some comment explanation. rather hard to parse. for one thing you compare dim > 0, yet set value 1:-1 above so why not use those 2 values ?